packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -88,8 +95,18 @@ export const PaymentErrorView = ({
         </div>
 
         <div className="footer" data-testid="footer">
-          <ActionButton />
-          <TermsAndPrivacy plan={plan} />
+          {/* This error code means the subscription was created successfully, but
+          there was an error loading the information on the success screen. In this
+          case, we do not want a "Try again" or "Manage subscription" button. */}
+          {error.code !== 'fxa_fetch_profile_customer_error' ? (
+            <ActionButton data-testid={'error-view-action-button'} />
+          ) : null}
+          <PaymentLegalBlurb provider={undefined} />
+          <TermsAndPrivacy
+            showFXALinks={isPasswordlessCheckout}
+            plan={plan}
+            contentServerURL={config.servers.content.url}
+          />
```

Comment:
For this error case where the user subscribes successfully, but we fail to load the success page, what kind of button could we have here?

We probably don’t want the "Try again" label, since the sub was created successfully, and since they haven't set a password, they aren't logged in and couldn't go to "Manage subscriptions".

Give this, I propose no action button for this error case, but I am open to suggestions.

Reply:
Agreed.  The only thing we want the new customer to do is to check their email at this point.